tox.ini

```diff
@@ -12,6 +12,14 @@ commands =
         --doctest-modules \
         {posargs:scrapy_spider_metadata tests}
 
+[testenv:min]
+base_python = python3.8
```

Comment:
Is it ok to support 3.8+? https://github.com/scrapinghub/scrapinghub-entrypoint-scrapy declares 3.6 support. It seems we should either 

* stop supporting old Pythons in https://github.com/scrapinghub/scrapinghub-entrypoint-scrapy, or
* start supporting 3.6 here, or
* make scrapy-spider-metadata optional in scrapinghub-entrypoint-scrapy, so that it's only used in recent Pythons (@cc @wRAR )

Reply:
Option 3 sounds good to me, gives users an incentive to upgrade to Python versions that still get security updates, and keeps the Scrapy Cloud stack lighter (we don’t need to add scrapy-spider-metadata to the stack if we don’t want to, we can instead ask users to include it in their requirements).